### PR TITLE
Update data-move-faq.md

### PR DESCRIPTION
**Pull request by andybergen**
SFB and signup changes.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/data-move-faq.md [hunk 1]
```diff
@@ -32,40 +32,35 @@ Core customer data is a term that refers to a subset of customer data defined in
 - SharePoint Online site content and the files stored within that site
 - Files uploaded to OneDrive for Business 
 
+## What is in scope for Teams migration?
+
+In addition to Exchange Online, SharePoint Online, and OneDrive for Business; Microsoft will migrate Teams data to the local datacenter.  
+- Teams chat messages, including private messages and channel messages. 
+- Teams images used in chats. 
+
+Teams files are stored in SharePoint Online and Teams chat files are stored in OneDrive for Business.  Voicemail, calendar, chat history, and contacts are stored in Exchange Online.  In many cases, Exchange Online, SharePoint Online and OneDrive for Business are already used by the customer in the local datacenter geo and are also part of the Microsoft 365 migration program for eligible customer countries.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/data-move-faq.md:41]:
```suggestion
Teams files are stored in SharePoint Online and Teams chat files are stored in OneDrive for Business.  Voicemail, calendar, chat history, and contacts are stored in Exchange Online. In many cases, Exchange Online, SharePoint Online, and OneDrive for Business are already used by the customer in the local datacenter geo and are also part of the Microsoft 365 migration program for eligible customer countries.
```

File: microsoft-365/enterprise/data-move-faq.md [hunk 2]
```diff
@@ -32,40 +32,35 @@ Core customer data is a term that refers to a subset of customer data defined in
...
+
 ## At what point is my migration complete so that my tenant's core customer data is being stored at rest in my new geo?
 
 Due to shared dependencies between Exchange Online and SharePoint Online/OneDrive for Business, any migration cannot be considered 
-completed until both services are migrated.  Exchange Online and SharePoint Online/OneDrive for Business often migrate at separate times and independently from one another.  Tenant admins receive confirmation in Message Center when each service migration is completed and can view the data location card in the Admin Center at any time to confirm the core customer data at rest location for 
+completed until both services are migrated.  Exchange Online and SharePoint Online/OneDrive for Business often migrate at separate times and independently from one another.  Customer tenant admins receive confirmation in Message Center when each service migration is completed and can view the data location card in the Admin Center at any time to confirm the core customer data at rest location for
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/data-move-faq.md:46]:
```suggestion
completed until both services are migrated.  Exchange Online and SharePoint Online/OneDrive for Business often migrate at separate times and independently from one another. Customer tenant admins receive confirmation in Message Center when each service migration is completed and can view the data location card in the Admin Center at any time to confirm the core customer data at rest location for 
```

File: microsoft-365/enterprise/data-move-faq.md [hunk 3]
```diff
@@ -32,40 +32,35 @@ Core customer data is a term that refers to a subset of customer data defined in
...
+
  ## Where is my core customer data located?
 
-Tenant admins can view the data location card in the Admin Center at any time to confirm the core customer data at rest location for 
+Customer tenant admins can view the data location card in the Admin Center at any time to confirm the core customer data at rest location for 
 each service, specifically for their tenant.  We also publish the location of datacenter geos, datacenters, and location of Office 
-365 customer data on the [Office 365 interactive datacenter maps ](https://office.com/datamaps) as a reference for the current default core customer data at rest locations for new tenants.  You can verify the location of your customer data at rest via the Data Location section under your Organization Profile in the Microsoft 365 Admin Center.  
+365 customer data on the [Microsoft 365 interactive datacenter maps ](https://office.com/datamaps) as a reference for the current default core customer data at rest locations for new tenants.  You can verify the location of your customer data at rest via the Data Location section under your Organization Profile in the Microsoft 365 Admin Center.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/data-move-faq.md:63]:
```suggestion
365 customer data on the [Microsoft 365 interactive datacenter maps](https://office.com/datamaps) as a reference for the current default core customer data at rest locations for new tenants. You can verify the location of your customer data at rest via the Data Location section under your Organization Profile in the Microsoft 365 Admin Center.  
```

File: microsoft-365/enterprise/data-move-faq.md [hunk 4]
```diff
@@ -81,12 +76,11 @@ It is not possible for us to remove you from the process after you submit your r
...
   
-We may be able to accept requests on an exception basis to grant your tenant a committed deadline to complete the move. Please 
-contact [Microsoft 365 Support](https://go.microsoft.com/fwlink/p/?LinkID=522459) to make the request.
+We cannot accept requests for migration after the open enrollment period.
 
 ## What if I want to move my data in order to get better network performance?
   
-Physical proximity to a Microsoft 365 datacenter is not a guarantee for a better networking performance. There are many factors and components that impact the network performance between the end-user and the Microsoft 365 service. For more information about this and performance tuning see [Network planning and performance tuning for Microsoft 365](network-planning-and-performance.md).
+Physical proximity to a Microsoft 365 datacenter is not a guarantee for a better networking performance. There are many factors and components that affect the network performance between the end-user and the Microsoft 365 service. For more information about this and performance tuning see [Network planning and performance tuning for Microsoft 365](network-planning-and-performance.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/data-move-faq.md:83]:
```suggestion
Physical proximity to a Microsoft 365 datacenter is not a guarantee for a better networking performance. There are many factors and components that affect the network performance between the end-user and the Microsoft 365 service. For more information about this and performance tuning, see [Network planning and performance tuning for Microsoft 365](network-planning-and-performance.md).
```

File: microsoft-365/enterprise/data-move-faq.md [hunk 5]
```diff
@@ -136,9 +130,11 @@ Yes. Microsoft maintains a large global network with public Internet connections
 
 ## My tenant has configured the [Multi Geo add-on](https://aka.ms/multi-geo). Can I still enroll in my tenant in the Microsoft 365 Move Program to change my default geo and move any user not in a satellite region to the new default geo?
 
-Yes, your tenant is eligible to enroll. We will move all EXO mailboxes from your current default geo to your new local datacenter geo.  We will not move any EXO mailboxes configured in Multi Geo satellite regions to continue to respect satellite region data residency as you’ve intended.  
+Yes, your tenant is eligible to enroll but there are significant considerations as tenant-level move is not fully supported for customers that have configured Multi-Geo.
+
+SharePoint Online and OneDrive for Business cannot migrate to the new datacenter geo at the tenant level through this program.  The customer administrator can configure OneDrive for Business shares to move to any available region using Multi-Geo, but the default location for the tenant cannot be changed once Multi-Geo has been configured for a tenant.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/data-move-faq.md:135]:
```suggestion
SharePoint Online and OneDrive for Business cannot migrate to the new datacenter geo at the tenant level through this program. The customer administrator can configure OneDrive for Business shares to move to any available region using Multi-Geo, but the default location for the tenant cannot be changed once Multi-Geo has been configured for a tenant.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - a few minor edits.

**Pull request merged**